nncf/quantization/algorithms/min_max/onnx_backend.py

```diff
@@ -73,40 +68,96 @@ def target_point(target_type: TargetType,
         return ONNXTargetPoint(target_type, target_node_name, port_id)
 
     @staticmethod
-    def create_activation_quantizer_insertion_command(target_point: ONNXTargetPoint,
+    def create_activation_quantizer_insertion_command(nncf_graph: NNCFGraph,
+                                                      target_point: ONNXTargetPoint,
                                                       quantizer_config: QuantizerConfig,
                                                       statistics: MinMaxTensorStatistic) \
-            -> ONNXQuantizerInsertionCommand:
-        axis = 1 if quantizer_config.per_channel else None
-        parameters = calculate_activation_quantizer_parameters(statistics, quantizer_config, axis)
-        return ONNXQuantizerInsertionCommand(target_point, parameters)
+                                                      -> ONNXQuantizerInsertionCommand:
+        return ONNXMinMaxAlgoBackend._create_quantizer_insertion_command(nncf_graph,
+                                                                         target_point,
+                                                                         quantizer_config,
+                                                                         statistics)
 
     @staticmethod
-    def create_weight_quantizer_insertion_command(target_point: ONNXTargetPoint,
+    def create_weight_quantizer_insertion_command(nncf_graph: NNCFGraph,
+                                                  target_point: ONNXTargetPoint,
                                                   quantizer_config: QuantizerConfig,
-                                                  weight_tensor: np.ndarray,
-                                                  node: NNCFNode) -> ONNXQuantizerInsertionCommand:
-        axis = node.metatype.weight_definitions.weight_channel_axis if quantizer_config.per_channel else None
-        parameters = calculate_weight_quantizer_parameters(weight_tensor, quantizer_config, axis)
+                                                  statistics: MinMaxTensorStatistic) -> ONNXQuantizerInsertionCommand:
+        return ONNXMinMaxAlgoBackend._create_quantizer_insertion_command(nncf_graph,
```

Comment:
Why not place this method in the `command_creation.py`?

Reply:
Because `_create_quantizer_insertion_command` uses function `_get_axis` which is used during statistics collector builder.